docs/_docs/windows.md

```diff
@@ -3,11 +3,107 @@ title: Jekyll on Windows
 permalink: /docs/windows/
 ---
 
+[WSL-Guide]: https://docs.microsoft.com/en-us/windows/wsl/install-win10
+[MSFT-Store-Ubuntu]: https://www.microsoft.com/en-us/p/ubuntu/9nblggh4msv6
+[MSFT-Store-Debian]: https://www.microsoft.com/en-us/p/debian-gnu-linux/9msvkqc78pk6
+[BASH-WSL]: https://msdn.microsoft.com/en-us/commandline/wsl/about
+[No-Sudo]: [https://jekyllrb.com/docs/troubleshooting/#no-sudo
+
 While Windows is not an officially-supported platform, it can be used to run Jekyll with the proper tweaks. This page aims to collect some of the general knowledge and lessons that have been unearthed by Windows users.
 
+You can either install Jekyll natively or use Windows Subsystem for Linux (aka. Bash on Windows).
+
+## Installation via Bash on Windows 10
+
+If you are using Windows 10 version 1607 or later, another option to run Jekyll is by [installing][WSL-Guide] the Windows Subsystem for Linux.  You can do this by installing [Ubuntu](MSFT-STORE-Ubuntu), [Debian](MSFT-STORE-Debian), or your preferred distro from the Microsoft Store.  The instructions below using `apt` and thus are for Debian-based distros.
+
+Once installed, run your distro of choice from the start menu.
+
+First, we need to update our systems' packages:
+
+```sh
+sudo apt-get update -y && sudo apt-get upgrade -y
+```
+
+Now we can install Ruby. To do this we will use a repository from [BrightBox](https://www.brightbox.com/docs/ruby/ubuntu/), which hosts optimized versions of Ruby for Ubuntu.
+
+```sh
+sudo apt-add-repository ppa:brightbox/ruby-ng
+sudo apt-get update
+sudo apt-get install ruby2.4 ruby2.4-dev build-essential dh-autoreconf
+```
+
+Next, [configure gem][No-Sudo] so that packages are installed in your $HOME directory so `sudo` is not required.  Add the following to the bottom of your .bashrc file:
```

Comment:
"gem" here should be in backticks so that it's clear that you mean the _command_